test/ui/cypress/support/auth.js

```diff
@@ -0,0 +1,51 @@
+import { verifyRequiredEnvVariables } from './../e2e/environment_helpers'
+
+Cypress.Commands.add('loginWithRHAccount', () => {
+   verifyRequiredEnvVariables()
+
+    cy.visit(Cypress.env("DEPLOYMENT_DRIVER_URL"), {
+          retryOnStatusCodeFailure: true,
+          retryOnNetworkFailure: true,
+          timeout: 5000,
+    })
+
+    //cy.get('#pf-login-username-id').type('admin') - hardcoded by Deployment Engine UI
```

Comment:
There is way too much this one command. Let's split it up into two commands:
1. One for logging in to the deployment driver engine with `admin` and a password
2. One for doing SSO

With such split, we will later be able to use this test suite for both testing during development and at verification/gating testing.

Reply:
@tznamena , in which situation you want to bypass Red Hat account login for deployment engine UI's testing? Did you mean we need to add some extra logic to control if we need the Red Hat Account login for development testing? For QE's testing, Red Hat account login is needed.

Reply:
We need to stop thinking about the separation between development and QE and start thinking about collaboration on the same code base.
Having said that, all that I'm suggesting is to split the command into two commands.

Reply:
Agree! It is better that both developers and QE use the same test suite. One question here: if we split it into two commands, how can we get to the "Red Hat Account login" command without the steps that are needed for the Deployment Engine UI? We can't locate the Red Hat account login dialog without the command of Deployment Engine UI login. I think it is better to combine them together instead of splitting them into two commands.

Reply:
As per the discussion with @tznamena , I will keep them as they are. @tznamena will submit a new PR to refine them.